### ISSUE TITLE
text flicker in ie8

### ISSUE THREAD

**Issue by youngwind** [arv/ExplorerCanvas#10]
I use this excanvas in IE8 for Chart.js ployfill, but the text flicker and sometimes display when i moverover the canvas as below.
Why?
How to fix it?
![demo](https://cloud.githubusercontent.com/assets/8401872/13392329/aa17933e-df14-11e5-8fd8-77f089558620.gif)

**Comment by thejae** [comment]
Are you running a virtual machine on Mac OS X? If so, what environment is this?

**Comment by youngwind** [comment]
Yes, that's right. At that time, I run a virtual machine in VMWare with window7. @thejae
